memory/arena_test.cc

```diff
@@ -127,10 +127,10 @@ static void ApproximateMemoryUsageTest(size_t huge_page_size) {
     usage = arena.ApproximateMemoryUsage();
   }
   if (huge_page_size) {
-    ASSERT_TRUE(usage > mem_usage ||
+    ASSERT_TRUE(usage >= mem_usage ||
```

Comment:
Double-check if we need these two changes, after checking ApproximateMemoryUsage

Reply:
Right, gone.